src/ScorecardInterface.js

```diff
@@ -0,0 +1,47 @@
+let scorecard = new Scorecard();
```

Comment:
Feedback from Arthur on interface

Interface looks great
Readme super clear as always
Tests all clear and passing
Pins remaining do not sum to 10
Can keep scoring past 10 frames
10th frame does not input bonus rolls in interface
Sums on interface for perfect game should be 30 not 20
Interface gives a ‘frame 11’ before gameover
Gameover displayed before 12th strike entered if 3 strikes in final frame.